docs/integrations/ember-and-json-api.md

```diff
@@ -38,12 +38,40 @@ You will also want to set the `key_transform` to `:unaltered` since you will adj
 ActiveModelSerializers.config.key_transform = :unaltered
 ```
 
-Lastly, in order to properly handle JSON API responses, we need to register a JSON API renderer, like so:
+In order to properly handle JSON API responses, we need to register a JSON API renderer, like so:
 
 ```ruby
 # config/initializers/active_model_serializers.rb
 require 'active_model_serializers/register_jsonapi_renderer'
 ```
+Rails also requires your controller to tell it that you accept and generate JSONAPI data.  To do that, you use `respond_to` in your controller handlers.  You can add `ActionController::MimeResponds` to your application controller to enable this:
```

Comment:
why is this? was this mentioned in the slack convo you had with @bf4? might be a good idea to state *why* you *need* respond_to.